AnimatedSprite.lua

```diff
@@ -472,4 +472,30 @@ end
 
 function AnimatedSprite:update()
 	self:updateAnimation()
-end
\ No newline at end of file
+end
+
+--Extra helper functions
+
+--Gets a state by name, so data like tick step can be changed dinamically, for example, in the situation of changing the speed of a spinning top
+---@return table
+function AnimatedSprite:getState(name)
+	return self.states[name]
+end
+
+--gets the current state
+---@return table
```

Comment:
Comments use EmmyLua style, so three `-` is required
```suggestion
---Get the current state
---@return table state Reference to the current state
```